### PR TITLE
Do not instrument built-ins

### PR DESCRIPTION
**Pull request by peterschrammel**
fixes https://github.com/diffblue/cbmc/issues/259

### CODE REVIEW

File: src/goto-instrument/cover.cpp [hunk 1]
```diff
@@ -1375,9 +1390,10 @@ void instrument_cover_goals(
   Forall_goto_functions(f_it, goto_functions)
   {
     if(f_it->first==ID__start ||
-       f_it->first=="__CPROVER_initialize")
+       f_it->first=="__CPROVER_initialize" ||
```
Comment (tautschnig) [inline src/goto-instrument/cover.cpp]: Use `CPROVER_PREFIX` "initialize" (and `#include <util/cprover_prefix.h>`) [yes, there are ~2000 other locations where that should get fixed]

File: src/goto-programs/remove_returns.cpp [hunk 2]
```diff
@@ -185,7 +185,7 @@ void remove_returnst::do_function_calls(
           t_a->code=code_assignt(function_call.lhs(), rhs);
           t_a->function=i_it->function;
 
-          // fry the previous assignment
+          // try the previous assignment
```
Comment (tautschnig) [inline src/goto-programs/remove_returns.cpp]: I do not think that was a typo!?

**Comment by tautschnig** [comment]
I think this PR might need some rework as we've now got a different way of determining built-ins.

**Comment by peterschrammel** [comment]
@tautschnig, fixed.

**Comment by tautschnig** [comment]
@peterschrammel Would you mind rebasing to get all fixes from master such as to repair the AppVeyor failure? The irep_ids merge teaches me that we should have all tests succeed before accepting. Thanks!

**Comment by tautschnig** [comment]
It seems a large number of tests have blank lines. I'm working on a PR to remedy this.

**Comment by peterschrammel** [comment]
How are tests with blank lines related to this PR?

**Comment by peterschrammel** [comment]
Rebasing...

**Comment by tautschnig** [comment]
The blank lines weren't related - current master is/was still broken...

**Comment by peterschrammel** [comment]
@tautschnig, all green now.

**Pull request merged**